rusk-wallet/CHANGELOG.md

```diff
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [0.2.0] - 2022-02-04
+
+### Added
+- Default to UDS transport [#520]
```

Comment:
This is a breaking change (the previous version connect to tcp_ip as default)

Maybe we should mention the new feature in the "Added" version and specify the default behaviour in the "Changed" one

Reply:
It is indeed a breaking change and I bumped the version accordingly, I can add it to the CHANGELOG too, no prob.